src/main/scala/org/hammerlab/guacamole/commands/jointcaller/Input.scala

```diff
@@ -16,10 +17,11 @@ import org.hammerlab.guacamole.readsets.{SampleId, SampleName}
  * @param analyte rna or dna
  */
 case class Input(index: SampleId,
-                 sampleName: SampleName,
```

Comment:
we should just call this class `JointCallerInput` instead of having multiple `Input` classes in the code base